### PR TITLE
update studio manage schema

### PR DESCRIPTION
**Pull request by foesa-yang**

### CODE REVIEW

**Review by whitewum — APPROVED** [review]

File: docs-2.0/nebula-studio/manage-schema/st-ug-crud-index.md [hunk 1]
```diff
@@ -37,16 +30,20 @@ To create an index on the **Schema** page, follow these steps:
 
    a. **Index Type**: Choose to create an index for a tag or for an edge type. In this example, **Edge Type** is chosen.
 
-   b. **Name**: Choose a tag name or an edge type name. In this example, **action** is chosen.
+   b. **Name**: Choose a tag name or an edge type name. In this example, **follow** is chosen.
+
+   c. **Index Name**: Specify a name for the new index. In this example, **follow_index** is used.
 
-   c. **Index Name**: Specify a name for the new index. In this example, **action_index** is used.
+   d. **Indexed Properties**: Click **Add**, and then, in the dialog box, choose a property. If necessary, repeat this step to choose more properties. You can drag the properties to sort them. In this example, `degree` are chosen.
```
Comment (randomJoe211) [inline docs-2.0/nebula-studio/manage-schema/st-ug-crud-index.md:37]:
```suggestion
   d. **Indexed Properties**: Click **Add**, and then, in the dialog box, choose a property. If necessary, repeat this step to choose more properties. You can drag the properties to sort them. In this example, `degree` is chosen.
```

File: docs-2.0/nebula-studio/manage-schema/st-ug-crud-index.md [hunk 2]
```diff
@@ -37,16 +30,20 @@ To create an index on the **Schema** page, follow these steps:
...
 
-   d. **Indexed Properties**: Click **Add**, and then, in the dialog box, choose a property. If necessary, repeat this step to choose more properties. You can drag the properties to sort them. In this example, `actionId` and `label` are chosen.
-   !!! note
+    !!! note
 
-       The order of the indexed properties has an effect on the result of the `LOOKUP` statement. For more information, see [nGQL Manual](https://docs.nebula-graph.io/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/lookup-syntax/#error_code_411 "Click to go to the Nebula Graph website").
+        The order of the indexed properties has an effect on the result of the `LOOKUP` statement. For more information, see [nGQL Manual](../../3.ngql-guide/7.general-query-statements/5.lookup.md "Click to go to the Nebula Graph website").
+
+   e. **Comment**: The remarks of a certain property or the index itself. The maximum length is 256 bytes. By default, there will be no comments on a index, `follow_index` is used.
```
Comment (randomJoe211) [inline docs-2.0/nebula-studio/manage-schema/st-ug-crud-index.md:43]: "By default, there will be no comments on a index, `follow_index` is used."
Something wrong with the logic of this sentence. Please double-check it.
And it should be "no comments on an index".

File: docs-2.0/nebula-studio/manage-schema/st-ug-crud-space.md [hunk 3]
```diff
@@ -28,25 +23,29 @@ To create a graph space on the **Schema** page, follow these steps:
 2. On the **Graph Space List** page, click the **+ Create** button.
 3. On the **Create** page, do these settings:
 
-   a. **Name**: Specify a name to the new graph space. In this example, `basketballplayer` is used. The name must be distinct in the database.
+   a. **Name**: Specify a name to the new graph space. In this example, `basketballplayer` is used. The name must be distinct in the database. The name cannot be used keywords or reserved keywords as identifiers. For more information, see [keywords](../../3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md).
+
+   b. **Vid type**: The data types of VIDs are restricted to `FIXED_STRING(<N>)` or `INT64`; a graph space can only select one VID type, `FIXED_STRING(32)` is used. For more information, see [VID](../../1.introduction/3.vid.md).
+
+   c. **Comment**: The remarks of a certain property or the space itself. The maximum length is 256 bytes. By default, there will be no comments on a space, `Statistics of basketball players` is used.
```
Comment (randomJoe211) [inline docs-2.0/nebula-studio/manage-schema/st-ug-crud-space.md:30]: Same issue. Why "there will be no comments on a space" then "`Statistics of basketball players` is used"? What is their connection?

**Review by randomJoe211 — CHANGES_REQUESTED** [review]

**Review by whitewum — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

File: docs-2.0/nebula-studio/manage-schema/st-ug-crud-space.md [hunk 4]
```diff
@@ -28,25 +23,29 @@ To create a graph space on the **Schema** page, follow these steps:
...
 
-![The Create page with settings for a graph space](../figs/st-ug-026-1.png)
+![The Create page with settings for a graph space](../figs/st-ug-026.png)
 
 ## Delete a graph space
 
 !!! danger
 
-    Deleting the image space will delete all the data in it, and the deleted data cannot be restored if it is not [backed up](../../7.data-security/3.manage-snapshot.md).
+    Deleting the Space will delete all the data in it, and the deleted data cannot be restored if it is not [backed up](../../7.data-security/3.manage-snapshot.md).
```
Comment (whitewum) [inline docs-2.0/nebula-studio/manage-schema/st-ug-crud-space.md:48]: space

**Review by whitewum — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Review by izhuxiaoqing — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**